src/jit/hwintrinsicxarch.cpp

```diff
@@ -265,6 +366,15 @@ bool Compiler::compSupportsHWIntrinsic(InstructionSet isa)
                                                     isFullyImplmentedISAClass(isa));
 }
 
+static bool is64bitOnlyIntrinsicOn32bit(HWIntrinsicCategory category, var_types type)
```

Comment:
This name is also confusing to me. How about a more general name, e.g. `isTypeSupportedForIntrinsic`?